content/english/supply-chain-analytics/ComingSoonPython

```diff
@@ -0,0 +1,13 @@
+---
+title: "Python- Extra Challenge" 
```

Comment:
Can you make this an .md file?

Reply:
Yes. For now I just wanted it to be a draft for any of my notes until I'm ready to build out the Python content.